packages/pluginutils/src/normalizePath.ts

```diff
@@ -1,9 +1,7 @@
-import { win32, posix } from 'path';
-
 import type { NormalizePath } from '../types';
 
 const normalizePath: NormalizePath = function normalizePath(filename: string) {
-  return filename.split(win32.sep).join(posix.sep);
```

Comment:
going to have to disagree with this change. while that regex is valid, the explicitness of using `win32.sep` and `posix.sep` has semantic value for future maintenance. if you'd like to build your regex using those constants, I would approve the PR.

Reply:
Thanks for your advice, now it's using `win32.sep` and `posix.sep`.